addons/account/models/account.py

```diff
@@ -1219,7 +1218,13 @@ def create_invoice_from_attachment(self, attachment_ids=[]):
             attachment.write({'res_model': 'mail.compose.message'})
             invoice.message_post(attachment_ids=[attachment.id])
             invoices += invoice
+        return invoices
 
+    def create_invoice_from_attachment(self, attachment_ids=None):
+        ''' Create the invoices from files.
+         :return: A action redirecting to account.move tree/form view.
+        '''
+        invoices = self.generate_invoice_from_attachment(attachment_ids)
```

Comment:
since `attachment_ids` is an optional field,
```suggestion
        invoices = self.generate_invoice_from_attachment(attachment_ids=attachment_ids)
```